examples/java8/src/main/java/org/apache/beam/examples/MinimalWordCountJava8.java

```diff
@@ -56,15 +59,15 @@ public static void main(String[] args) {
     Pipeline p = Pipeline.create(options);
 
     p.apply(TextIO.Read.from("gs://apache-beam-samples/shakespeare/*"))
+
      .apply(FlatMapElements
          .into(TypeDescriptors.strings())
-         .via((String word) -> Arrays.asList(word.split("[^a-zA-Z']+"))))
+         .via((String word) -> Arrays.asList(word.split(TOKENIZER_PATTERN))))
```

Comment:
We can keep the regex inline, instead of defining a new variable. I think this makes more sense for a minimal example.